### DISCUSSION TITLE
Open second Dropdown instantly instead of closing the first one before

### DISCUSSION THREAD

**Discussion by gopeter** [radix-ui/primitives#1345] [Help] [answered]
I have two Dropdowns side by side. Once one is open and I want to open the other one, a click on the trigger of the second one doesn't open it immediately but closed the open Dropdown at first, so I need two clicks. Is there any option I may have missed to configure this? Thanks for your help 🙏

**Comment by andy-hook** [comment, marked as answer]
Hi @gopeter , sounds like you want to adjust the [modality](https://www.radix-ui.com/docs/primitives/components/dropdown-menu#root). You can set `modal={false}` on `DropdownMenu.Root` to achieve this.

**Reply by gopeter** [reply]
Indeed! Thanks a lot. I've went through each prop, but I didn't got that this one would have this effect.

**Comment by djalmaaraujo** [comment]
@andy-hook I just want to say that this is probably the easiest solution to have a Dropdown triggering a Dialog. If you set the DropDown as modal=false, you can correctly have the focus trapped into the Dialog and not on the Dropdown anymore.

Related issue: https://github.com/radix-ui/website/issues/384

**Reply by andy-hook** [reply]
Hey @djalmaaraujo I'm not sure how this relates to the original question? or do you mean to say that adjusting the modality is useful in others cases as well?

**Reply by djalmaaraujo** [reply]
@andy-hook Yes. I am trying to do dropdown+dialog to work with the proper focus management, and this looks like a simple and elegant solution. It says to dropdown to let it go and let the modal use the focus.
